### ISSUE TITLE
They entered my account sold my items and spent the money from the items sold

### ISSUE THREAD

**Issue by Tiozeus123** [ValveSoftware/Dota2-Gameplay#6834]
Today I entered my account in the afternoon (1:00 p.m.) and I realized that they had sold my items and had bought 1 item that does not cost more than S/.0.05 to S/.106.32.
there are more than 80 items that he sold from my account without my knowledge
The mobile steam guard did not notify me nor did it withhold the items

**Comment by nelsonnyan2001** [comment]
Yeah... I'm sure you've been super careful with your passwords and your mobile authenticator and it's totally a dota 2 related bug since you seem to be the only person on earth experiencing this!

**Comment by Iceapple331** [comment]
> Today I entered my account in the afternoon (1:00 p.m.) and I realized that they had sold my items and had bought 1 item that does not cost more than S/.0.05 to S/.106.32. there are more than 80 items that he sold from my account without my knowledge The mobile steam guard did not notify me nor did it withhold the items

I am also facing this problem. Someone used my account and bought an item from Dota 2

**Comment by andraip** [comment]
This is the Dota 2 bugtracker, not Steam support.
